### PR TITLE
Popup Window

### PR DESCRIPTION
**Pull request by degisew**
### Popup Window

**In this project I did the following features:**
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- I implemented popups for both mobile and desktop screen sizes.

### CODE REVIEW

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @degisew, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a pair programming project with no contributions from your partner.  Kindly report any issue you had with your partner to the student's success team and they'll permit you to work solo or look for a suitable partner.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Comment by degisew** [comment]
Hi @topeogunleye, I already reported about my partner's absence to the student success team and they make my solo. now, I am in a solo so, please review my project

**Comment by degisew** [comment]
Hi @topeogunleye, I already reported about my partner's absence to the student success team and they make my solo. now, I am in a solo so, please review my project
Microverse report

File: style.css [hunk 1]
```diff
@@ -33,6 +33,124 @@
   display: none;
 }
 
+#popup-detail-window {
```
Comment (vickymarz) [inline style.css:36]: - [ ] Good job adding the popup 👍 . Kindly ensure that the popup on the mobile view fits properly to the screen viewport to match your chosen Figma design. The popup is currently compressed and distorted.


| Figma Design  | Your Design |
| ------------- | ------------- |
| image | image  |

File: index.html [hunk 2]
```diff
@@ -8,19 +8,27 @@
...
     <script src="mobileMenu.js" defer></script>
+    <script src="popup.js" defer></script>
   </head>
 
   <body>
+      <div id="popup-detail-window">
+        <div id="popup-element-container">
+          
+        </div>
+      </div>
```
Comment (vickymarz) [inline index.html:19]: - [ ] Good job centering the popup in the page vieewport👍 . Please the spaces beside the popup on the desktop view is too large so i would recommend you increase the width of the popup to match your chosen Figma design. 


| Figma Design  | Your Design |
| ------------- | ------------- |
| image) |image  |

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @degisew , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Popup working properly ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by degisew** [comment]
Great review. Thank you. I will walk through it.

File: desktop.css [hunk 3]
```diff
@@ -57,13 +57,44 @@ body {
...
+}
+
+.popup-image {
+  display: flex;
+
+  /* width: 100%; */
+  margin: 15px;
+}
+
+.popup-image img {
```
Comment (mwanawabangona) [inline desktop.css:71]: - [ ] `optional`
- [ ] Kindly make sure the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., `font`, `text`, `margins` and the `background blur`)

`your template`
desktopzim

`figma template`
figma pop-up

File: desktop.css [hunk 4]
```diff
@@ -57,13 +57,44 @@ body {
   color: black;
 }
 
+#popup-element-container {
+  margin-left: 60px;
+}
+
+.popup-image {
```
Comment (mwanawabangona) [inline desktop.css:64]: - [ ] `optional`
- [ ] Kindly make sure all the `mobile ppup cards` follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (`font`, `text`, `margins`)
`your design`
mmm

`figma template`
mobile pf

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @degisew ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**